runtime/sak.c

```diff
@@ -51,19 +59,23 @@
      `L"C:\\OCaml\xd83d\xdc2b\\lib"`
  */
 
-void usage(void)
+static void usage(void)
 {
   printf(
     "OCaml Build System Swiss Army Knife\n"
     "Usage: sak command\n"
     "Commands:\n"
     " * encode-C-literal path - encodes path as a C string literal\n"
+    " * show-includes <cfile> <target>"
+      " - transforms MSVC /showInclude for Make inclusion\n"
+    "   cl /nologo /Zs /showIncludes /c myfile.c | "
+         "sak show-includes myfile.c myfile.obj > myfile.d\n"
   );
 }
 
 /* Converts the supplied path (UTF-8 on Unix and UCS-2ish on Windows) to a valid
-   C string literal. On Windows, this is always a wchar_t* (L"..."). */
-void encode_C_literal(char_os *path)
+   C string literal. On Windows, this is always a wchar_t* (T"..."). */
```

Comment:
T"" isn't right - it'd be T(), but there's no need for this change... T() matters if you're targetting Win9x 🙂

Reply:
that's a sed gone wrong